src/main/java/createQuiz/CreateQuizInteracter.java

```diff
@@ -1,4 +1,36 @@
 package createQuiz;
 
-public class CreateQuizInteracter {
+import entities.Deck;
+import entities.Quiz;
+
+import java.util.List;
+
+public class CreateQuizInteracter implements CreateQuizInputBoundary{
+
+    private final CreateQuizOutputBoundary outputBoundary;
+
+    public CreateQuizInteracter(CreateQuizOutputBoundary outputBoundary) {
+        this.outputBoundary = outputBoundary;
+    }
+
+    @Override
+    public CreateQuizOutputData createQuiz(CreateQuizInputData inputData) {
+        String quizName = inputData.getQuizName();
+        List<Deck> quizDecks = inputData.getQuizDecks();
+
+        String message;
```

Comment:
Small nitpick: To clean up this code section, you could remove lines 21 and 22. Instead of

String message;
...
message = "This quiz name exists already. Failed to create new quiz";


You could do the following directly on lines 25, 26, 29, 31

String message = "This quiz name exists already. Failed to create new quiz";


This is just how I code and how I find it best to keep code small and organized. You could follow this style or keep to your own, I just want to let you know of this alternative.